cypress/e2e/flow-builder/connections/creating-removing.cy.ts

```diff
@@ -0,0 +1,41 @@
+describe('Creating a connection', () => {
+  beforeEach(() => {
+    cy.createFlow({
+      label: 'Creating a connection',
+    })
+
+    cy.addBlock(['Content', 'Message']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`).as('firstBlock')
+    })
+
+    cy.addBlock(['Content', 'Numeric Response']).then(blockId => {
+      cy.get(`[data-cy="block--${blockId}"]`).as('secondBlock')
+    })
+
+    // wait for flow validation after adding blocks
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(300)
+  })
+
+  it('creates and removes a connection', () => {
+    cy.get('@firstBlock').find('[data-cy^="exit--"]').as('exit')
+    cy.get('@secondBlock').find('[data-cy^="block-handle--"]').as('secondBlockArea')
+
+    // deselect blocks and close block editor
+    cy.get('[data-cy="builder-canvas"]').click('left')
+
+    cy.get('@exit').dragAndDropTo('@secondBlockArea')
+
+    // The only way to identify current line so far: https://github.com/anseki/leader-line/issues/185
+    cy.get('body>.leader-line:last-of-type').as('connection')
+
+    cy.get('@connection').should('be.visible')
+
+    // remove connection
+    cy.get('@exit')
+      .realHover()
+      .click()
+
+    cy.get('@connection').should('not.exist')
```

Comment:
@bulatgab , thanks for providing this test for the connections. I think, the remaining tests is on the undo/redo: Making sure we can undo and redo connections actions

- [ ] address this

Reply:
my suggestion is just to add few test after this line
```
cy.undo() // to undo the remove connection
// then assert something

cy.undo() // to undo the add connection with drag-n-drop
// assert something

cy.redo() // put back the connection
// then assert something

cy.redo() // remove back the connection
// then assert something
```